scaleway/resource_lb_beta.go

```diff
@@ -123,6 +123,12 @@ func resourceScalewayLbBetaRead(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
+	if _, ok := d.GetOkExists("ip_id"); ok {
+		_ = d.Set("release_ip_on_deletion", false)
```

Comment:
Should set the value contained in the state (I'm not 100% sure this is even required to do)